rules/sprint-review-retro-email/rule.md

```diff
@@ -68,53 +68,52 @@ Attendees: *(Optional as they may be in the to and CC)*
 | 24112  | Integrate Business Logic to MVC app | Done      | 8          |
 | 24097  | Styling                             | Committed | 16         |
 
-**Figure: Sprint Backlog from {{Link to Sprint Backlog in Azure DevOps}}** 
+**Figure: Sprint Backlog from {{Link to Sprint Backlog in Azure DevOps}}**
 
 1. Sprint Burndown (a quick overview of the Sprint)
 
 ![Figure: Sprint Burndown](burndown.JPG)
 
-2. Code Coverage (hopefully tests are increasing each Sprint)
+1. Code Coverage (hopefully tests are increasing each Sprint)
    XXX
-3. Velocity *(Optional)*
+1. Velocity *(Optional)*
    XXX
-4. Burnup (for the release - the whole project, how are we tracking for the big picture?)
+1. Burnup (for the release - the whole project, how are we tracking for the big picture?)
 
 ![Figure: Release Burnup](ReleaseBurnup.jpg)
 
-5. Build Pipeline Health & Production Deployments (How many times did we deploy to Production?)
+1. Build Pipeline Health & Production Deployments (How many times did we deploy to Production?)
 
    ![Figure: Build Pipeline Health from DevOps](thumbnail_image.png)
 
 ![Figure: Deployments from {{ DEPLOYMENT SERVICE }}](production-deploy.png)
 
-6. Application Health Overview Timeline (For the entire Sprint)
+1. Application Health Overview Timeline (For the entire Sprint)
 
 ![Application Health Overview Timeline.png](ApplicationInsights.jpg)
 
-7. Product Roadmap
+1. Product Roadmap
```

Comment:
Looks like these numbers should still be 1-7?

Reply:
In markdown, having any numbers means it's an ordered list. It's generally easier/better (spending on the use case of course) to use 1 for each bullet as when you re-order there is less admin in renumbering